### ISSUE TITLE
Empty filename in literal packet after the file signing

### ISSUE THREAD

**Issue by ni4** [riboseinc/rnp#198]
Running:
`rnp --sign hello.txt`
Produces the following literal packet:

```
=> PGP_PARSER_PTAG: Literal Data

==== ptag new_format=1 type=11 length_type=0 length=0x13 (19) position=0xf (15)
Literal Data packet
=> CT: Literal Data Header
LITERAL DATA HEADER packet
  literal data header format=b filename=''
    modification time=0 (Thu Jan  1 00:00:00 1970)
```

It would be reasonable to include file's name and modification date into this packet.

**Comment by ronaldtse** [comment]
@ni4 thanks for reporting this, this is definitely a bug. Would you be able to help fix it? Thanks!

**Comment by ni4** [comment]
Sure thing, I'll do that.

**Comment by ronaldtse** [comment]
Thanks @ni4 , seems that modification time is empty too, it'll be great you can handle that while you're at it :-)

**Comment by ni4** [comment]
Fixed in 7ca4eb6
